### PR TITLE
[WIP]Feature/readme

### PR DESCRIPTION
**Pull request by dinhl**
Please help me check README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-# Specifications
-## Download
-### GET
-### /api/file-management/file?loc={}
+This provides file management service which includes the following:
+  - Downloading
+  - Uploading
+  - Deleting
+## Prerequisites
+- Installed Pycharm with Python 3.6
```
Comment (dinhl) [inline README.md]: @faraonc please read comment

File: README.md [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
-# Specifications
-## Download
-### GET
-### /api/file-management/file?loc={}
+This provides file management service which includes the following:
+  - Downloading
+  - Uploading
+  - Deleting
+## Prerequisites
+- Installed Python 3.7
```
Comment (faraonc) [inline README.md]: consider removing the word "Installed"

File: README.md [hunk 3]
```diff
@@ -1,21 +1,41 @@
...
+- Installed Webstorm
```
Comment (faraonc) [inline README.md]: remove word "Installed"

File: README.md [hunk 4]
```diff
@@ -1,21 +1,41 @@
...
-### /api/file-management/file?name={}&type={}
+## How to install Dependencies
+ TO-DO
 
-## Delete
-### DELETE
-### /api/file-management/file?loc={}
+# How to run service
+- After having a contract with API
+- Develop a new service for hwsc-file-svc (e.g., Download, Upload, Delete)
```
Comment (faraonc) [inline README.md]: Put another step and write TO-DO

**Comment by faraonc** [comment]
closing in favor of https://github.com/faraonc/hwsc-file-transaction-svc/pull/8

**Pull request closed without merging**